README.md

```diff
@@ -5,38 +5,99 @@
 
 This project is a modern implementation of the Ubuntu Desktop installer, using [subiquity](https://github.com/canonical/subiquity) as a backend and [Flutter](https://flutter.dev/) for the UI.
 
-## Checkout & Run
+![Preview Screenshot](.github/docs/images/preview_screenshot.png)
 
-Grab the source:
+## Availability
+
+The new Ubuntu Desktop installer is *planed* to ship with the next Ubuntu Desktop LTS 22.04 Impish Idri.
+
+## Test the new Ubuntu Desktop installer
+
+If you want to test the new installer before 22.04 goes live, you can download a preview canary image below and install it for example in a virtual machine:
+
+[Download](http://cdimage.ubuntu.com/impish/daily-canary/current/)
+
+
+## Building and running manually
+
+To build the installer manually you need to have the [Flutter UI toolkit](https://flutter.dev) installed and the source code in this repository downloaded on your computer.
+
+To install Flutter you have two options: a manual installation by cloning the flutter repo and adding it tou your $PATH or installing the Flutter snap.
+
+### Either: Install Flutter manually
+
+Install the Flutter dependencies:
 ```sh
-git clone https://github.com/canonical/ubuntu-desktop-installer.git
-cd ubuntu-desktop-installer
-git submodule update --init --recursive
+sudo apt install git curl cmake meson clang cmake pkg-config libgtk-3-dev
 ```
-
-Install and configure Flutter:
+Install Flutter:
 ```sh
-snap install flutter --classic
+mkdir ~/development
+cd ~/development
+git clone https://github.com/flutter/flutter.git -b master
+echo 'export PATH="$PATH:$HOME/development/flutter/bin"' >> ~/.bashrc
+```
+
+### Or: Install Flutter with Snapd
+
+```
+sudo snap install flutter --classic
+```
```

Comment:
I'd suggest removing all this and relying on the installation instructions provided by the Snap Store and the Flutter docs.